### ISSUE TITLE
Add player informations in /heroes/${heroId}/players endpoint

### ISSUE THREAD

**Issue by bdbch** [odota/core#1646]
Currently we don't have that many data coming from the players endpoint in the heroes route. Could we potentially load additional data from the player endpoint so we can build more unified react components for player preview cards etc.?

https://github.com/odota/core/blob/master/routes/spec.js#L3469

See here

**Comment by howardchung** [comment]
yes we can, should just be adding additional columns to the SELECT part of the query.  What data did you want to show?  team affiliation probably, and I think we can display player image as well (that only requires account_id I think)

**Comment by bdbch** [comment]
I would like to have the same data as the other endpoints that are associated with player data. For example:

```
avatar
last_login
name
personaname
rank_tier
score
```

Main purpose for me would be a unified component for players with a "player" tooltip similar to the tooltips on Facebook, Github, etc. like this one:

![image](https://user-images.githubusercontent.com/6538827/39668213-99b6141c-50c7-11e8-8786-f8174f818523.png)

**Comment by howardchung** [comment]
The thing is that these are pro players and many of these fields may not be applicable.  e.g. their avatar is probably not of interest and their personaname is just whatever they have set on Steam (which can be kind of random).

**Comment by bdbch** [comment]
Ah so https://www.opendota.com/heroes/102/players

here only pro players are listed? I just wondered because I would like to have one unified variant for the player profiles in tables because right now they are very mixed up.

They can look like this:
![grafik](https://user-images.githubusercontent.com/6538827/39704666-5be9eff0-520c-11e8-89bb-3930caae64f3.png)

Or like this:
![grafik](https://user-images.githubusercontent.com/6538827/39704681-64a01b1a-520c-11e8-98e1-6399c329bc85.png)

I'd rather have them all displayed by the same component.

**Comment by howardchung** [comment]
Yes, only pro players are listed here.  However there are some differences in what we want to display, e.g. avatar should probably be replaced with a photo of the player, and their rank is probably not needed.  Up to you if you want to combine them into the same component.
